source/upgrade/v3.txt

```diff
@@ -203,4 +203,11 @@ Version 3.0 Breaking Changes
     property instead.
 
 - The driver removes individual cluster events from ``MongoClient.Cluster``. To listen for
-  cluster events, use `ClusterBuilder.Subscribe() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.Core.Configuration.ClusterBuilder.Subscribe.html>`__.
\ No newline at end of file
+  cluster events, use `ClusterBuilder.Subscribe() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.Core.Configuration.ClusterBuilder.Subscribe.html>`__.
+
+- The driver seals all concrete serializers. To implement a custom serializer, implement the
```

Comment:
We had some questions of "why" classes got sealed.
It's worth adding a general comment followed by these concrete examples:
Something like "Some of the types that were not designed for extension via inheritance became sealed. Starting with 3.0 we recommend using "composition over inheritance" approach for extending the driver functionality."